catalog/app/containers/Bucket/Summarize.tsx

```diff
@@ -659,3 +664,62 @@ export function SummaryNested({ handle, mkUrl, packageHandle }: SummaryNestedPro
     _: () => null,
   })
 }
+
+const useConfigureAppearanceStyles = M.makeStyles((t) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    padding: t.spacing(2, 0),
+  },
+  button: {
+    '& + &': {
+      marginLeft: t.spacing(1),
+    },
+  },
+}))
+
+interface ConfigureAppearanceProps {
+  hasReadme: boolean
+  hasSummarizeJson: boolean
+  packageHandle: PackageHandle
+  path: string
+}
+
+export function ConfigureAppearance({
+  hasReadme,
+  hasSummarizeJson,
+  packageHandle,
+  path,
+}: ConfigureAppearanceProps) {
+  const classes = useConfigureAppearanceStyles()
+  const readme = FileEditor.useAddFileInPackage(
+    packageHandle,
+    join(path || '', 'README.md'),
+  )
+  const summarize = FileEditor.useAddFileInPackage(
+    packageHandle,
+    join(path || '', 'quilt_summarize.json'),
+  )
+  return (
+    <div className={classes.root}>
+      {!hasSummarizeJson && (
+        <StyledTooltip title="Clicking the button opens the file editor. After saving, you'll see a package creation dialog to add a new quilt_summarize.json file.">
+          <RRDom.Link to={summarize} className={classes.button}>
+            <M.Button color="primary" size="small" variant="outlined">
+              Add quilt_summarize
+            </M.Button>
+          </RRDom.Link>
+        </StyledTooltip>
+      )}
+      {!hasReadme && (
+        <StyledTooltip title="Clicking the button opens the file editor. After saving, you'll see a package creation dialog to add a new README file.">
```

Comment:
```suggestion
        <StyledTooltip title="Open the editor to author a README file. Upon saving, a package revision dialog will show up, letting you add that file to the package.">
```
maybe smth like this?

Reply:
Yes, it's better